flink-connectors/flink-sql-connector-pulsar/pom.xml

```diff
@@ -69,6 +69,7 @@ under the License.
 									<include>org.bouncycastle:bcprov-jdk15on</include>
 									<include>org.bouncycastle:bcutil-jdk15on</include>
 									<include>org.slf4j:jul-to-slf4j</include>
+									<include>javax.xml.bind:jaxb-api</include>
```

Comment:
FYI: flink-dist bundles jaxb-api

Reply:
(so it shouldn't be necessary for connectors to bundle it as well)

Reply:
And _if_ you bundle it it should be done in such a way that it is only loaded from the jar on Java 11, as it is done for flink-dist.

Reply:
> FYI: flink-dist bundles jaxb-api already

If flink-dist bundles the jaxb-api. Why we would meet a class not found error on flink ci?
https://dev.azure.com/apache-flink/apache-flink/_build/results?buildId=31702&view=logs&j=6e8542d7-de38-5a33-4aca-458d6c87066d&t=5846934b-7a4f-545b-e5b0-eb4d8bda32e1&l=15537

Reply:
Presumably because it is missing outside of the container, i.e., by the pulsar environment.

Reply:
Yeah, I add the `jaxb-api` only for e2e tests now.